ql/src/java/org/apache/hadoop/hive/ql/ddl/table/AbstractBaseAlterTableAnalyzer.java

```diff
@@ -98,6 +100,7 @@ protected void addInputsOutputsAlterTable(TableName tableName, Map<String, Strin
 
       if (AlterTableUtils.isFullPartitionSpec(table, partitionSpec)) {
         // Fully specified partition spec
+        SessionState.getSessionConf().set(Context.operationNameConf, op.getName());
```

Comment:
can we get the operation from Context.getOperation?

Reply:
Removed checking of operation type.